src/lib/Zikula/Core/Forms/ZikulaCsrfProvider.php

```diff
@@ -11,12 +11,12 @@
 
 namespace Zikula\Core\Forms;
 
-use Symfony\Component\Form\Extension\Csrf\CsrfProvider\CsrfProviderInterface;
+use Symfony\Component\Security\Csrf\CsrfTokenManagerInterface;
 
 /**
  * Csrf provider based on zikula csrf system.
  */
-class ZikulaCsrfProvider implements CsrfProviderInterface
+class ZikulaCsrfProvider implements CsrfTokenManagerInterface
```

Comment:
If we switch to the native one the custom implementation should be removed from 1.4 and 2.0-spec then. But as part of a separate issue.